CHANGES.md

```diff
@@ -9,8 +9,12 @@ To be released.
  - Added `Swarm.DifferentVersionPeerEncountered` event handler that can handle
    events when a different version of a peer is discovered.  [[#167]], [[#185]]
  - Added `Peer.AppProtocolVersion` property.  [[#185]]
+ - `Swarm.StartAsync()` now receives the height(`Index`) of the block from
+   other known peers and synchronizes the blocks if necessary before starts
+   another process. [[#187]]
```

Comment:
What does “another process” here mean?

Reply:
Receiving and processing messages(`Swarm.ReceiveMessageAsync()`) Repeating delta distribution(`Swarm.RepeatDeltaDistributionAsync()`), and so on.

Reply:
It would be better if that's elaborated, e.g.:

> … before propagating/receiving pinpointed recent blocks to prevent inefficient round-trips.